liquibase-core/src/main/java/liquibase/command/CommandDefinition.java

```diff
@@ -17,7 +17,7 @@
      */
     private final SortedSet<CommandStep> pipeline;
 
-    private final SortedMap<String, CommandArgumentDefinition<?>> arguments = new TreeMap<>();
+    private final Map<String, CommandArgumentDefinition<?>> arguments = new LinkedHashMap<>();
```

Comment:
I don't think changing this definition to be insertion order for the purposes of prompting is good. As we get to a better CommandStepPipline refactor, the insertion order will tend toward the more random feeling. Also, if/when other features are interested in the arguments in different orders than what the prompting requires we'll have conflicts.

I think it's best to keep this in the simple alphabetical order, and figure out a way to have the logic of the order in which to prompt be more orthogonal.

One idea: allow the interactivity logic to be able to say what configurations they depend on being set, then use DependencyUtil logic to dynamically figure out the order?

Reply:
DependencyUtil is a neat idea, I didn't know it existed. I will change this so we don't depend on insertion order.

Reply:
DependencyUtil turned out to be a bit overkill for this situation, when all I really need is a list. I went with the simpler solution for now.